lldb/source/Plugins/Process/minidump/ProcessMinidump.cpp

```diff
@@ -961,6 +993,73 @@ class CommandObjectProcessMinidumpDump : public CommandObjectParsed {
   }
 };
 
+class CommandObjectProcessMinidumpCheckMemory : public CommandObjectParsed {
+public:
+  CommandObjectProcessMinidumpCheckMemory(CommandInterpreter &interpreter)
+      : CommandObjectParsed(interpreter, "process plugin check-memory",
```

Comment:
@bulbazord Hey Alex, do you have any idea on a better name than `check-memory`? Thanks!